browser-test/src/applicant/applicant_application_index.test.ts

```diff
@@ -671,7 +671,7 @@ test.describe('applicant program index page', () => {
           await applicantQuestions.applyProgram(
             primaryProgramName,
             /* northStarEnabled= */ true,
-            /* isApplicationUnstarted= */ false,
+            /* showProgramOverviewPage= */ false,
```

Comment:
I updated this param name since this should be false for common intake, even when the application is unstarted.